packages/eslint-plugin/src/rules/method-signature-style.ts

```diff
@@ -156,39 +158,41 @@ export default createRule<Options, MessageIds>({
               context.report({
                 node: methodNode,
                 messageId: 'errorMethod',
-                *fix(fixer) {
-                  const methodNodes = [
-                    methodNode,
-                    ...duplicatedKeyMethodNodes,
-                  ].sort((a, b) => (a.range[0] < b.range[0] ? -1 : 1));
-                  const typeString = methodNodes
-                    .map(node => {
-                      const params = getMethodParams(node);
-                      const returnType = getMethodReturnType(node);
-                      return `(${params} => ${returnType})`;
-                    })
-                    .join(' & ');
-                  const key = getMethodKey(methodNode);
-                  const delimiter = getDelimiter(methodNode);
-                  yield fixer.replaceText(
-                    methodNode,
-                    `${key}: ${typeString}${delimiter}`,
-                  );
-                  for (const node of duplicatedKeyMethodNodes) {
-                    const lastToken = context.sourceCode.getLastToken(node);
-                    if (lastToken) {
-                      const nextToken =
-                        context.sourceCode.getTokenAfter(lastToken);
-                      if (nextToken) {
-                        yield fixer.remove(node);
-                        yield fixer.replaceTextRange(
-                          [lastToken.range[1], nextToken.range[0]],
-                          '',
-                        );
+                fix: skipFix
```

Comment:
Missing code coverage on this line can be achieved with the following test (or equivalent):

```ts
interface Foo {
  foo(): this;
  foo(): Promise<this>;
}
```